### PR TITLE
[PLT-6697] Added CTRL/CMD+SHIFT+K shortcut to open 'more direct messages' dialog

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Added CTRL/CMD+SHIFT+K shortcut to open 'more direct messages' dialog

#### Ticket Link
Jira ticket: [PLT-6697](https://mattermost.atlassian.net/browse/PLT-6697)

#### Checklist
- [x] Has UI changes

### CODE REVIEW

File: webapp/components/sidebar.jsx [hunk 1]
```diff
@@ -299,6 +305,9 @@ export default class Sidebar extends React.Component {
             ChannelActions.goToChannel(nextChannel);
             this.updateScrollbarOnChannelChange(nextChannel);
             this.isSwitchingChannel = false;
+        } else if (Utils.cmdOrCtrlPressed(e) && e.shiftKey && e.keyCode === Constants.KeyCodes.K) {
+            e.preventDefault();
```
Comment (jwilander) [inline webapp/components/sidebar.jsx]: No need to do `e.preventDefault();` here if you're doing it again in `handleOpenMoreDirectChannelsModal ()`

Comment (saturninoabril) [inline webapp/components/sidebar.jsx]: Got it! see update. thanks!

**Comment by saturninoabril** [comment]
updated PR - adding item 4 from Jira ticket:

4. Add the following text to /shortcuts output below CTRL/CMD+ALT+K team switcher shortcut
```{CTRL/CMD}+SHIFT+K: Open direct messages menu```

File: app/command_shortcuts.go [hunk 2]
```diff
@@ -37,7 +37,7 @@ func (me *ShortcutsProvider) GetCommand(T goi18n.TranslateFunc) *model.Command {
 }
 
 func (me *ShortcutsProvider) DoCommand(args *model.CommandArgs, message string) *model.CommandResponse {
-	shortcutIds := [28]string{
+	shortcutIds := [29]string{
```
Comment (hmhealey) [inline app/command_shortcuts.go]: Not something to change now, but we can change this to `[...]string{` at some point to have it autosize so we don't need to keep changing it ourselves :)

**Pull request merged**